pom.xml

```diff
@@ -19,7 +19,7 @@
         <springdoc-version>1.6.6</springdoc-version>
         <modelmapper-version>3.1.0</modelmapper-version>
         <jackson-datatype-jsr310-version>2.13.1</jackson-datatype-jsr310-version>
-        <flyway-version>8.5.13</flyway-version>
+        <flyway-version>9.22.3</flyway-version>
```

Comment:
it works for pgFlex 15 vers too ?

Reply:
From 9.0.5 Flyway is compatible with PostgreSQL 15:
https://documentation.red-gate.com/fd/release-notes-for-flyway-engine-179732572.html
![image](https://github.com/user-attachments/assets/e73ae15c-4664-4aa1-a5a8-b15f2d9d3a78)

Abou PGFLEX: "The compatibility includes PostgreSQL-specific features and extensions such as custom schema handling and advanced SQL functions. However, Flyway itself does not explicitly manage extensions like pgFlex. To use such extensions, you need to ensure they are installed and configured in your PostgreSQL instance prior to running Flyway migrations​"